hibike/virtual_device.py

```diff
@@ -89,19 +105,22 @@ def main():
         if msg.get_message_id() in [hm.MESSAGE_TYPES["SubscriptionRequest"]]:
             # Update the delay, subscription time,
             # and params, then send a subscription response
-            print("Subscription request recieved")
+            if args.verbose:
```

Comment:
We can just have a function called `log` that contains the `args.verbose` logic, and then use `log("MESSAGE")` instead.